src/Wavefront.AspNetCore.SDK.CSharp/Wavefront.AspNetCore.SDK.CSharp.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="App.Metrics.AspNetCore" Version="2.0.0" />
     <PackageReference Include="Wavefront.OpenTracing.SDK.CSharp" Version="2.0.0" />
+    <PackageReference Include="Wavefront.SDK.CSharp" Version="1.6.0" />

```

Comment:
Let's add a TODO comment here to remove this once OpenTracing SDK version has been updated.